mmpose/codecs/simcc_label.py

```diff
@@ -22,8 +22,8 @@ class SimCCLabel(BaseKeypointCodec):
 
     Args:
         input_size (tuple): Input image size in [w, h]
-        simcc_type (str): The SimCC label type. Options are ``'gaussian'`` and
-            ``'one-hot'``. Defaults to ``'gaussian'``
+        simcc_type (str): The SimCC label smoothing strategy. Options are
```

Comment:
What about using a more specific argument name like 'smoothing_type' or 'smoothing_mode'?

Reply:
OK, I think `smoothing_type` is better